src/api/exemptions/controllers/submit-exemption.integration.test.js

```diff
@@ -0,0 +1,189 @@
+import {
+  describe,
+  it,
+  expect,
+  beforeAll,
+  afterAll,
+  beforeEach,
+  vi
+} from 'vitest'
+import { ObjectId } from 'mongodb'
+import { EXEMPTION_STATUS } from '../../../common/constants/exemption.js'
+import {
+  createCompleteExemption,
+  mockCredentials
+} from '../../../../tests/test.fixture.js'
+
+describe('POST /exemption/submit', () => {
+  let server
+  let db
+  let exemptionId
+  let mockDate
+
+  beforeAll(async () => {
+    const { createServer } = await import('../../../server.js')
+    server = await createServer()
```

Comment:
There's a [test helper in the frontend repo](https://github.com/DEFRA/marine-licensing-frontend/blob/main/tests/integration/shared/test-setup-helpers.js#L45-L58) to make it simple for integration tests to set up the server, which reduces non-test code in the integration test files. Could use something similar in the backend?